src/main/java/com/example/bookstoreapp/controller/AuthenticationController.java

```diff
@@ -13,14 +16,21 @@
 
 @RequiredArgsConstructor
 @RestController
-@RequestMapping("/auth")
+@RequestMapping("/api/auth")
```

Comment:
add `server.servlet.context-path=/api` to application.properties

```suggestion
@RequestMapping("/auth")
```